### ISSUE TITLE
After package push, deliver string that points to the package version instead of the files UI

### ISSUE THREAD

**Issue by JacksonMaxfield** [quiltdata/quilt#1531]
I (and team) am loving the new `quilt3 catalog` we have a minor nitpick which is that after we push a package the string that is returned is:

```
Package aics/some_package@f8e6363 pushed to s3://allencell-internal-quilt
Run `quilt3 catalog s3://allencell-internal-quilt/aics/some_package/` to browse.
```

Which opens to the what I refer to as the Files UI or the Files Tab, but if we just pushed a package version then that may be super confusing and instead it would be great to be able to open straight up to what I refer to as the Package UI or the Package Version View.

Something like:
```
Package aics/some_package@f8e6363 pushed to s3://allencell-internal-quilt
Run `quilt3 catalog s3://allencell-internal-quilt/packages/aics/some_package/tree/1581029480/` to browse.
```

I feel like the reason this wasn't done was because the >1000 files behavior is still happening. If that is the case then I understand waiting to change this over.

**Comment by akarve** [comment]
Yeah, the reason we did this is that package view truncates after 1K files. In any case, If you set your `quilt3.config(navigator_url="https://my-catalog")` then it'll print package URL out (instead of the S3 URL).
